SumofAllNumbers.py

```diff
@@ -0,0 +1,19 @@
+def sum_all(num):
+	first = num[0]
+	second = num[1]
+
+	if(first > second):
+		temp = first
```

Comment:
did you know in python you can actually swap values like this:

```py
a = 5
b = 10
a,b = b,a
# a = 10
# b = 5
```

pretty neat!